cleverhans_tutorials/tutorial_models_tfe.py

```diff
@@ -113,11 +113,3 @@ def get_params(self):
   def get_layer_names(self):
     """:return: the list of exposed layers for this model."""
     return self.layer_names
-
-  def call(x):
```

Comment:
Why was this deleted here?

Reply:
- It's not used
- It's always been broken (no "self") argument
- It's not even the right name (should be "__call__")
- The right thing (__call__) method is implemented in the base class, so this class shouldn't implement it anyway

Reply:
SGTM